src/plugins/foxPage/components/FoxyAssetMarketData.tsx

```diff
@@ -0,0 +1,41 @@
+import { AssetId } from '@shapeshiftoss/caip'
+import { foxyAddresses } from '@shapeshiftoss/investor-foxy'
+import { useFoxy } from 'features/defi/contexts/FoxyProvider/FoxyProvider'
+import { useCallback, useEffect, useState } from 'react'
+import { AssetMarketData } from 'components/AssetHeader/AssetMarketData'
+import { selectAssetById } from 'state/slices/selectors'
+import { selectMarketDataById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type FoxyAssetMarketDataProps = {
+  assetId: AssetId
+}
+
+export const FoxyAssetMarketData = ({ assetId }: FoxyAssetMarketDataProps) => {
```

Comment:
```suggestion
export const FoxyAssetMarketData: React.FC<FoxyAssetMarketDataProps> = ({ assetId }) => {
```

Reply:
I removed this component which is now replaced by hook `useFoxyMarketData`